modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/loader/utils/ImplicitObjectUtils.java

```diff
@@ -35,13 +42,27 @@ private ImplicitObjectUtils() {}
    */
   public static ResolverSet buildImplicitResolverSet(ParameterizedModel parameterizedModel, MuleContext muleContext) {
     ResolverSet resolverSet = new ResolverSet();
+
+    Map<String, Object> parameterValues = new HashMap<>();
+    for (ParameterModel model : parameterizedModel.getAllParameterModels()) {
```

Comment:
don't do this here. Add a constructor/factory method on ParametersResolver which does this from a ParmeterizedModel